pytealutils/math/test_fixed_point.py

```diff
@@ -0,0 +1,23 @@
+from pyteal import *
+
+from tests.helpers import assert_output, logged_int, logged_bytes
+
+from .fixed_point import *
+
+
+def test_fixed_point():
+    fp = FixedPoint(64, 8)
+    _fp = FixedPoint(64, 16)
+
+    fp1 = fp.wrap(2.3)
+    fp2 = fp.wrap(3.3)
```

Comment:
I think having random tests' inputs could improve tests' robustness and coverage.

Reply:
started to setup for this but need to think about how to populate valid tests and deal with large python integers/floats